EIPS/eip-3672.md

```diff
@@ -0,0 +1,72 @@
+---
+eip: 3672
+title: Offchain resolution for ENS
+author: Nick Johnson (@arachnid), 0age (@oage)
+discussions-to: <URL>
+status: Draft
+type: Standards Track
+category: ERC
+created: 2021-07-21
+requires: 137, 3668
+---
+
+## Simple Summary
+Specifies an updated resolution process for ENS that supports offchain resolution and wildcard lookups.
+
+## Abstract
+Instead of using the resolution process from EIP 137, clients should start with the provided name and check successively shorter suffixes until they find a name with a resolver. Then, they should use the process described in EIP 3668 to query the resolver.
```

Comment:
```suggestion
Instead of using the resolution process from [EIP-137](./eip-137.md), clients should start with the provided name and check successively shorter suffixes until they find a name with a resolver. Then, they should use the process described in [EIP-3668](./eip-3668) to query the resolver.
```

Reply:
If possible, I think it's better for this info (e.g. how this EIP relates to other things) to be in the motivation and leave the abstract as strictly focused on what this EIP does.